spartan/aztec-network/files/config/setup-service-addresses.sh

```diff
@@ -81,17 +81,25 @@ fi
 
 if [ "${PROVER_BROKER_EXTERNAL_HOST}" != "" ]; then
     PROVER_BROKER_ADDR="${PROVER_BROKER_EXTERNAL_HOST}"
-elif [ "${NETWORK_PUBLIC}" = "true" ]; then
-    PROVER_BROKER_ADDR=$(get_service_address "prover-broker" "${PROVER_BROKER_PORT}")
 else
     PROVER_BROKER_ADDR="http://${SERVICE_NAME}-prover-broker.${NAMESPACE}:${PROVER_BROKER_PORT}"
 fi
 
+# Configure OTEL_COLLECTOR_ENDPOINT if not set in values file
+if [ "${OTEL_COLLECTOR_ENDPOINT}" = "" ]; then
+    OTEL_COLLECTOR_PORT=${OTEL_COLLECTOR_PORT:-4318}
+    OTEL_COLLECTOR_ENDPOINT="http://metrics-opentelemetry-collector.metrics:$OTEL_COLLECTOR_PORT"
+fi
```

Comment:
I think this gets set every time regardless of `{{ Values.telemetry.enabled }}` flag. Not abig deal.